apps/backend/build.gradle.kts

```diff
@@ -31,4 +31,8 @@ dependencies {
     implementation(libs.ktor.server.config.yaml)
     testImplementation(libs.ktor.server.test.host)
     testImplementation(libs.kotlin.test.junit)
-}
\ No newline at end of file
+}
+
+tasks.named<JavaExec>("run") {
+    systemProperties["ktor.environment"] = env.MODE.orElse("prod")
```

Comment:
When the environment variable `MODE` is renamed, don't forget to rename it here as well!